src/kokoro_model.h

```diff
@@ -21,6 +21,15 @@ static std::map<char, std::string> KOKORO_LANG_TO_ESPEAK_ID = {
 	{'z', "sit/cmn"}
 };
 
+// A list of all of the top level GGUF names under kokoro.duration_predictor that have quantization compatible tensors. 
+const std::vector<std::string> DURATION_PREDICTOR_QUANTIZATION_COMPATIBLE_PARTS = {
```

Comment:
This should be initialized in a .cpp file to not break ODR.

Even better, make it a `static constexpr std::array<const char *, 5>` after moving it there.